mock-interactions.js

```diff
@@ -40,7 +40,10 @@
       bubbles: true,
       cancelable: true,
       clientX: xy.x,
-      clientY: xy.y
+      clientY: xy.y,
+      // Make this a primary input.
+      buttons: 1, // http://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons
+      which: 2    // http://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which
```

Comment:
`which` is used in a fallback case in `IronButtonState`, when `buttons` is not supported, which hopefully can go away soon.

In either case, `which` can't be set on a MouseEvent either with `new` or `initMouseEvent`, so we should just remove it.